content/faq.md

```diff
@@ -2,25 +2,27 @@
 title = "Frequently Asked Questions (FAQ)"
 +++
 
+**This article has been verified for VCMI 1.1 (release date 23.12.2022)**
+
 * Again, what is VCMI?
 
-> In simple words: Remake of Heroes of Might and Magic 3. We use original Heroes 3 graphic, sound and text assets and make new game that is supposed to work 100% same as H3. Everything in VCMI is made from scratch - loading game files, drawing graphics, AI, game mechanics etc. We also plan supporting WoG mod features, but for now there are areas in VCMI that need more attention than WoG features.
+> In simple words: Remake of Heroes of Might and Magic 3. We use original Heroes 3 graphic, sound and text assets and make new game that is supposed to work 100% same as H3. Everything in VCMI is made from scratch - loading game files, drawing graphics, AI, game mechanics etc. We also plan supporting WoG, HD mod and other cool features, but there are other areas that need our attention as well.
```

Comment:
I don't think WoG is any point of focus right now. Some features were developed years ago, and then it stopped. No scripting whatsoever and 90% of WoG are complex and weird scripts.

Reply:
Not current priority, but one of original goals was to remake WoG and I see VCMI scripting support as way to push more WoG stuff forward when time comes

Reply:
Proposing to edit last sentence:
"We also plan to add support for popular features from WoG and HD mod as well as some totally new features, but there are other areas that need our attention as well."

We won't "support" WoG/HD, as in, you won't be able to install then in vcmi. There is also no guarantee that we decide to implement *all* features from them.